apps/modernization-ui/src/apps/page-builder/page/management/edit/content/PageContent.tsx

```diff
@@ -1,14 +1,34 @@
-import { PagesTab } from 'apps/page-builder/generated';
+import { PageQuestionControllerService, PagesTab } from 'apps/page-builder/generated';
 import { Sections } from '../section/Sections';
 import { PageSideMenu } from './PageSideMenu';
 import styles from './page-content.module.scss';
+import { authorization } from 'authorization';
+import { usePageManagement } from '../../usePageManagement';
+import { useAlert } from '../../../../../../alert';
 
 type Props = {
     tab: PagesTab;
 };
 export const PageContent = ({ tab }: Props) => {
-    const handleAddQuestion = (subsection: number) => {
-        console.log('add question not yet implemented', subsection);
+    const { showAlert } = useAlert();
+    const { page } = usePageManagement();
+    const handleAddQuestion = (subsection: number, questionId: number) => {
+        const request = {
+            subsectionId: subsection,
+            questionId: questionId
+        };
+        PageQuestionControllerService.addQuestionToPageUsingPost({
+            authorization: authorization(),
+            page: page.id,
+            request
+        }).then((response: any) => {
+            showAlert({
+                type: 'success',
+                header: 'Add',
+                message: response.message || 'Add Question successfully on page'
+            });
+            return response;
```

Comment:
Issue(blocking): in usePageManagement there is a `fetch()` method that refetches the page so that once the add is complete the page is reloaded and the new question is displayed automatically without having to refresh the page